docs/learn/intro/obol-vs-others.md

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_position: 2
+description: Some of the key terms in the field of Distributed Validator Technology
+---
+
+# Obol vs Other DV Implementations
+
+This page outlines the unique features of Obol's DV implemenation, constrasting with other DV implementations. We built Obol’s DVT as a middleware to keep Ethereum secure, resilient, and composable. See also the blog article [Why We Built Charon as a Middleware](https://blog.obol.org/why-we-built-charon-as-a-middleware/).
+
+![Obol vs others table](/img/ObolvsOthers.png)
+
+## No private keys put on chain
+
+Obol's distributed key generation (DKG) event generates key shares for each node within the DV cluster. The entire validator key NEVER exists in one place. Keys are generated locally on the nodes, and can be backed up. The private keys of Obol DVs are NEVER uploaded to the internet or published on-chain.
+
+An alternative approach to doing this is to split it into shares, encrypt each share with the public key of a node operator, and publish the encrypted private key on chain. The operators’ node key could then decrypt the validator private key. In our opinion, this is not secure. We believe that the safest approach is to avoid the existence of a singular private key, and certainly never to post any private key to a public blockchain network.
+
+## Cluster independance: Clusters can upgrade independently
+
+DV clusters are independent from one another. They can run different versions of Charon, and don't need to upgrade together. Charon will NEVER require a hard fork or simultaneous update for any upgrades.  
+
+In an Obol DV cluster, nodes use LibP2P to communicate directly with each other, and communications are end-to-end encrypted with SSL. Clusters are independent and do not need to communicate with one another. When a new version of Obol’s Charon is released, Obol DV clusters can upgrade on their own time, individually from other DV clusters. 
+
+![Cluster Independence](/img/ClusterIndependence.png)
+
+## Cluster independance: No reliance on a single P2P gossip network
+
+In an Obol DV cluster, nodes use LibP2P to communicate directly with each other, and communications are end-to-end encrypted with SSL. This direct communication of nodes within a cluster improves latency, and makes cluster communications harder to attack with a denial of service (DOS) attack. It also allows an Obol DV cluster to be run within a private network. This may allow cost savings on data egress costs, for operators running cluster nodes across multiple locations of a single cloud provider, for example.
```

Comment:
```suggestion
In an Obol DV cluster, nodes use LibP2P to communicate directly with each other, and communications are end-to-end encrypted with TLS. This direct communication of nodes within a cluster improves latency, and makes cluster communications harder to attack with a denial of service (DOS) attack. It also allows an Obol DV cluster to be run within a private network. This may allow cost savings on data egress costs, for operators running cluster nodes across multiple locations of a single cloud provider, for example.
```

Reply:
Implementing via my own commit